### PR TITLE
add progress bar and BEGIN constant

### PR DESCRIPTION
**Pull request by jjanice**
Added disappearing progress bar for folks to watch while archiver is fetching data
Added BEGIN constant for 3/25 08:00 the beginning of commissioning to be used by spinners and xfel plot
Commented out debug messages and some code I think we don't need as a test... Seems ok so far.

### CODE REVIEW

File: arPullGlob.py [hunk 1]
```diff
@@ -507,12 +623,13 @@ def XfelDsply(strt):
...
     totalLo=[]
     totalHi=[]
+    BEGIN=datetime.dateime(2022,3,25,8,0,0)
     #print(matplotlib.__version__, numpy.__version__)
     #e,f,DisLow,DisHi, DisLoStd, DisHiStd = CmStats(datetime.time,datetime.time)
     #print "e=", e  , "\n\r", "DisLow=", DisLow, "DisLoStd=",DisLoStd
     #e, f, r1,r2,r3,r4  = cavStats("4","27",startDate,endDate)
     #print ("e=",e,"r1=", r1, r2) 
-    squid = XfelDsply(startDate)
+    squid = XfelDsply(BEGIN,startDate)
```
Comment (lisazacarias) [inline arPullGlob.py:632]: The variable names pain me...

File: arPullGlob.py [hunk 2]
```diff
@@ -446,49 +547,64 @@ def cavStats(cmUpper,cmLower, Start, End):
...
    # begin = datetime.date(strt.strftime('%Y,%m,%d'))
-    begin= datetime.date(int(2021),int(7),int(22))
-     
-#    print(begin)
+#    begin= datetime.date(int(2021),int(7),int(22))
+    print(BEGIN)
+
     aaaa=list(resultsIntlk.keys())
-    for n in range(0, 37):
+    for nn in range(0, 37):
```
Comment (lisazacarias) [inline arPullGlob.py:579]: All of these variable names...

**Review by lisazacarias — APPROVED** [review]
I really just want significantly more descriptive variable names hahahaha

File: arPullGlob.py (hunk 1 above)
Comment (jjanice) [reply arPullGlob.py:632]: I know. slowly but surely I'll rename them.

**Pull request merged**